client/components/nav.jsx

```diff
@@ -12,24 +16,40 @@ export default class Nav extends React.Component {
         text: 'Join Game'
       }
     ];
-    const { navOpen, handleClickNav } = this.props;
+    const loggedInLinks = [
+      {
+        href: '#home',
+        text: 'Home'
+      },
+      {
+        href: '#join',
+        text: 'Join Game'
+      }
+    ];
+    const links = !user.userId ? loggedOutLinks : loggedInLinks;
```

Comment:
`!userId ?` is really hard to understand. how about reversing this to eliminate the bang?